includes/transformer/class-post.php

```diff
@@ -883,43 +879,6 @@ protected function get_classic_editor_image_embeds( $max_images ) {
 		return $images;
 	}
 
-	/**
-	 * Get image attachments from the classic editor.
-	 * This is imperfect as the contained images aren't necessarily the
-	 * same as the attachments.
-	 *
-	 * @param int $max_images The maximum number of images to return.
-	 *
-	 * @return array The attachment IDs.
-	 */
-	protected function get_classic_editor_image_attachments( $max_images ) {
```

Comment:
let's keep that, for people that still use the classic editor

Reply:
Ah, this is only about the attachments 🤔

Reply:
Good to remove?

Reply:
That doesn't seem to be accounted for in lines 799-803, it only looks for `WP_HTML_Tag_Processor` and uses `get_classic_editor_image_embeds()` when it's available

Reply:
Oh, I see. I thought we were ditching all classic editor support with this, didn't read closely enough!

I think we should be good to get rid of it, then.